app/scripts/views/settings/emails.js

```diff
@@ -23,6 +23,8 @@ define(function (require, exports, module) {
   const EMAIL_REFRESH_SELECTOR = 'button.settings-button.email-refresh';
   const EMAIL_REFRESH_DELAYMS = 350;
 
+  const EMAIL_ENABLED_REGEX = /.+@mozilla\.com$|.+@restmail\.net$|.+@softvisioninc\.eu$|.+@softvision\.(com|ro)$/;
```

Comment:
I realize we are trying to move away from Able, but this is the sort of logic we've used Able for. This logic seems like it should live in fxa-content-experiments until we go through the full work of ripping it out.